docs/other-topics/dialect-specific-things.md

```diff
@@ -262,20 +254,9 @@ const sequelize = new Sequelize('MY_DSN', 'username', 'password', {
 
 The final connection string generated by the above configuration would look like `CMT=1;NAMING=0;...;DSN=MY_DSN;UID=username;PWD=password;`. Additionally, the `host` option will map the the `SYSTEM=` connection string key.
 
-## Data type: TIMESTAMP WITHOUT TIME ZONE - PostgreSQL only
```

Comment:
Modifying how `pg` parses values is not allowed, it would break our built-in data types. 

We make sure types are returned in a lossless representation, then it's up to them to modify the parsing using the static `parseDatabaseValue` method